src/features/create-token/steps/TokenomicsForm.test.tsx

```diff
@@ -0,0 +1,59 @@
+//- React Imports
+import React from 'react';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+
+//- Component Imports
+import { TokenomicsForm } from './TokenomicsForm';
+import { ZUIProvider } from '@zero-tech/zui/ZUIProvider';
+
+//- Type Imports
+import { TokenomicsFormSubmit } from '../CreateToken.types';
+
+describe('TokenomicsForm', () => {
+	test('Next button should not fire onSubmit when field values are invalid', async () => {
```

Comment:
The `invalid` and `valid` tests are only testing the "all valid" or "all invalid" states. We should probably also test that each value is validated - I imagine we can probably do this by mocking Formik.